android/MobiledgeXSDKDemo/app/src/main/java/com/mobiledgex/sdkdemo/qoe/QoeMapActivity.java

```diff
@@ -1078,4 +1086,40 @@ public ColoredPoint(LatLng coords, float dlSpeed, float upSpeed) {
             color = Color.parseColor(COLORS[score]);
         }
     }
+
+    @Override
+    public void onRegister() {
+
+    }
+
+    @Override
+    public void onVerifyLocation(AppClient.VerifyLocationReply.GPSLocationStatus status, double gpsLocationAccuracyKM) {
+
+    }
+
+    @Override
+    public void onFindCloudlet(AppClient.FindCloudletReply closestCloudlet) {
+
+    }
+
+    @Override
+    public void onGetCloudletList(AppClient.AppInstListReply cloudletList) {
+
+    }
+
+    @Override
+    public void showMessage(String text) {
+
+    }
+
+    @Override
+    public void showError(String text) {
+
+    }
+
+    @Override
+    public void getCloudlets(boolean clearExisting) {
+
+    }
```

Comment:
Do you still need this ?

Reply:
Oh, yeah. I need to rework that interface. This particular class only uses the MatchingEngine for getQosPositionKpi(). Nothing else.